cli_tools_tests/e2e/gce_ovf_import/test_suites/ovf_import_e2e_common.go

```diff
@@ -238,6 +245,22 @@ func VerifyInstance(instance *gcp.InstanceBeta, client daisyCompute.Client,
 		}
 	}
 
+	// Verify that the instance's tags match the user's request (regardless of order).
+	expectedTags := map[string]struct{}{}
+	for _, tag := range props.Tags {
+		expectedTags[tag] = struct{}{}
+	}
+	actualTags := map[string]struct{}{}
+	for _, tag := range instance.Tags.Items {
```

Comment:
This doesn't test if tags are also attached to transient VMs we use for inflation/translation. Our e2e tests are not set up to validate transient VMs so this would be difficult to implement. 

One way we could validate tags on transient VMs would be to simulate customer's setup that exposed this issue: have a VPC that is allowing traffic only from VMs that have a specific tag and use that VPC for the test.

Reply:
You're exactly right, and I suspect the approach you're describing of simulating the user's setup will be required.

In this PR, I wanted to focus on applying tags to the final instance/GMI; for the transient VMs, I'll do a follow-up.